### PR TITLE
Add AttemptWon event to the contract

### PR DESCRIPTION
**Pull request by vadkutsen**

### CODE REVIEW

File: contracts/Guess.sol [hunk 1]
```diff
@@ -78,7 +78,9 @@ contract Guess {
         require(msg.sender == owner, "Only owner can call this function."); 
         _;
     }
-    
+
+    event AttemptWon(address _winnerAccount, uint _amount);
```
Comment (tudor-malene) [inline contracts/Guess.sol:82]: can you make the winnerAccount as indexed.

**Review by tudor-malene — COMMENTED** [review]
thanks. One little change requested to make the event show up for the winner according to the Obscuro events design

File: contracts/Guess.sol (hunk 1 above)
Comment (vadkutsen) [reply contracts/Guess.sol]: Done

**Review by tudor-malene — APPROVED** [review]
LGTM!
Thanks @vadkutsen

**Pull request merged**